lib/cli_executor.rb

```diff
@@ -132,6 +132,13 @@ def self.get_version_for(user, host)
 
       unless res[:success]
         logger.debug res[:instruction]
+        if res[:stderr] =~ /Error from server \(ServiceUnavailable\)\: the server is currently unable to handle the request \(get projects\.project\.openshift\.io/
+          logger.info(">>>>>> Debug Segment: checking openshift-apiserver availability <<<<<<")
+          logger.info(res[:stderr])
+          adm_user = user.env.admin
+          debug_result = adm_user.cli_exec(:get, resource: 'co', resource_name: "openshift-apiserver")
```

Comment:
@zimo-xiao Only `oc get co openshift-apiserver`, less information shows us, to get more, try `oc describe co/openshift-apiserver | sed -n "/Status:/,$ p"`, that will be useful for us debugging.

Reply:
@wangke19 @zimo-xiao Can we also check KAS? or also  it is better to print also `oc get co` for all operators

Reply:
@zimo-xiao @gangwgr I think it would be nice to print all abnormal operators(instead of all operators) as part of debug action.
_oc get co | grep -v '.True.*False.*False'_

Reply:
The error `Error from server (ServiceUnavailable): the server is currently unable to handle the request (get projects.project.openshift.io) ` we captured is response by openshift-apiserver directly, 4.10 and later, `oc get co/openshift-apiserver` will display the reason of Degraded operator, but former releases don't have, we have to get by `oc describe co/openshift-apiserver | sed -n "/Status:/,$ p"`, at last we can check further by `oc get co | grep -v '.True.*False.*False'` just like @jmekkatt suggested.